pkg/cmd/attestation/verification/attestation.go

```diff
@@ -73,26 +73,43 @@ func loadBundlesFromJSONLinesFile(path string) ([]*api.Attestation, error) {
 
 	attestations := []*api.Attestation{}
 
-	scanner := bufio.NewScanner(file)
-	for scanner.Scan() {
-		b := scanner.Bytes()
+	reader := bufio.NewReader(file)
+
+	var line []byte
+	line, err = readLine(reader)
```

Comment:
```suggestion
	line, err = reader.ReadBytes('\n')
```

From the docs on https://pkg.go.dev/bufio#Reader.ReadLine:

>  Most callers should use [Reader.ReadBytes](https://pkg.go.dev/bufio#Reader.ReadBytes)('\n')